<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DiXCover</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QScrollArea" name="scrollArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="widgetResizable">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="scrollContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>593</width>
          <height>438</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="lblImage">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Load a image</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="propertiesWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTextBrowser" name="textBrowser">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QTableWidget" name="tableWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>150</number>
           </attribute>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>100</number>
           </attribute>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <column>
            <property name="text">
             <string>Property</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Value</string>
            </property>
           </column>
           <item row="0" column="0">
            <property name="text">
             <string>Title</string>
            </property>
           </item>
           <item row="1" column="0">
            <property name="text">
             <string>Author</string>
            </property>
           </item>
           <item row="2" column="0">
            <property name="text">
             <string>Publisher</string>
            </property>
           </item>
           <item row="3" column="0">
            <property name="text">
             <string>Date</string>
            </property>
           </item>
           <item row="4" column="0">
            <property name="text">
             <string>Description</string>
            </property>
           </item>
           <item row="5" column="0">
            <property name="text">
             <string>Location</string>
            </property>
           </item>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1012</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionPhoto_capture"/>
    <addaction name="actionVideo_capture"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionZoom_In_25"/>
    <addaction name="actionZoom_Out_25"/>
    <addaction name="actionNormal_Size"/>
    <addaction name="separator"/>
    <addaction name="actionFit_to_window"/>
   </widget>
   <widget class="QMenu" name="menuE_xtract">
    <property name="title">
     <string>E&amp;xtract</string>
    </property>
    <addaction name="actionMetadata_Identification"/>
   </widget>
   <widget class="QMenu" name="menuDevices">
    <property name="title">
     <string>Devices</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuDevices"/>
   <addaction name="menuE_xtract"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionPhoto_capture"/>
   <addaction name="actionVideo_capture"/>
   <addaction name="actionZoom_In_25"/>
   <addaction name="actionZoom_Out_25"/>
   <addaction name="actionNormal_Size"/>
   <addaction name="actionMetadata_Identification"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/images/icons/Photo.png</normaloff>:/images/icons/Photo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open image ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionZoom_In_25">
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/images/icons/Magnify-Glass-blue.png</normaloff>:/images/icons/Magnify-Glass-blue.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;In (+25%)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_Out_25">
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/images/icons/Reduce-Glass-orange.png</normaloff>:/images/icons/Reduce-Glass-orange.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;Out (-25%)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionNormal_Size">
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/images/icons/Adjust.png</normaloff>:/images/icons/Adjust.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Normal Size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionFit_to_window">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/images/icons/Ipad.png</normaloff>:/images/icons/Ipad.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Fit to Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionMetadata_Identification">
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/images/icons/Wizard.png</normaloff>:/images/icons/Wizard.png</iconset>
   </property>
   <property name="text">
    <string>Metadata &amp;Identification</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionVideo_capture">
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/images/icons/Movie.png</normaloff>:/images/icons/Movie.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Video capture ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionPhoto_capture">
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/images/icons/old-camera.png</normaloff>:/images/icons/old-camera.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Photo capture ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="app.qrc"/>
 </resources>
 <connections/>
</ui>
